src/clients/feed/feed.ts

```diff
@@ -392,31 +435,27 @@ class Feed {
   }
 
   private async makeStatusUpdate(itemOrItems: FeedItemOrItems, type: Status) {
-    // If we're interacting with an array, then we want to send this as a batch
-    if (Array.isArray(itemOrItems)) {
-      const itemIds = itemOrItems.map((item) => item.id);
-
-      return await this.apiClient.makeRequest({
-        method: "POST",
-        url: `/v1/messages/batch/${type}`,
-        data: { message_ids: itemIds },
-      });
-    }
+    // Always treat items as a batch to use the corresponding batch endpoint
+    const items = Array.isArray(itemOrItems) ? itemOrItems : [itemOrItems];
+    const itemIds = items.map((item) => item.id);
 
-    // Handle unx actions
-    if (type.startsWith("un")) {
-      return await this.apiClient.makeRequest({
-        method: "DELETE",
-        url: `/v1/messages/${itemOrItems.id}/${invertStatus(type)}`
-      });
-    }
```

Comment:
do we not need this anymore?

Reply:
nope, switched to only ever use the batch endpoint